### ISSUE TITLE
Error installing foundation-cli on Windows

### ISSUE THREAD

**Issue by jsmreese** [zurb/foundation-apps#203]
I see this error when I run `npm install -g foundation-cli` on Windows (Windows 7 64bit):

![image](https://cloud.githubusercontent.com/assets/1275528/5306343/91e16ed6-7bd1-11e4-89a0-6f72e00d26db.png)

**Comment by gakimball** [comment]
Can any Windows users help us test a new build of the CLI? We're working on improving error-handling and cross platform support.

Here's how to run the test install:

```
git clone https://github.com/zurb/foundation-cli.git
cd foundation-cli
npm install
npm test
```

If all goes well, you'll see three green check marks at the end of the install. If you get an error, please paste it in here so we can take a look. Thanks!

**Comment by tnedich** [comment]
Win 7 Ultimate 64-bit
io.js platform

> foundation-cli@1.0.4 test C:\Users\Admin\Desktop\zurb-test\foundation-cli
> ./bin/cli.js new test

'.' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! Test failed.  See above for more details.

**Comment by gakimball** [comment]
@tnedich Fixed that: 3d9b4737d80205311ac58364f4ea8ecc462b8acb

The command should at least start now.

**Comment by tnedich** [comment]
Win 7 Ultimate 64-bit
io.js platform

> ✗ Ruby gems not installed. Try running bundle manually.

**Comment by gakimball** [comment]
Hmm, well luckily we're going to remove Ruby from the install process entirely, so I won't stress about that. The Git, npm, and Bower parts working are the most critical.

**Comment by jgamao** [comment]
I am using Ubuntu 14.04 LTS in releasing my game but I also have this error.

I am also using nodejs.. Can you help me to solve this?

**Comment by gakimball** [comment]
@jgamao Which error are you getting? And is it with the current CLI or this test version?

**Comment by fhassan2014** [comment]
![image](https://cloud.githubusercontent.com/assets/8864393/6381089/8b454968-bd0c-11e4-9b3e-30dedcf7f73d.png)

**Comment by fhassan2014** [comment]
![error](https://cloud.githubusercontent.com/assets/8864393/6381120/c503e2e0-bd0c-11e4-8333-84bb038bbed2.jpg)

**Comment by gakimball** [comment]
@fhassan2014 Ah, sorry about that, there was a line of debugging code I left uncommented that forces an error to happen. If you have time, could you try testing it again? I'd appreciate it :)

- Delete the `test` folder the CLI created
- Run `git pull`
- `npm test` again

**Comment by fhassan2014** [comment]
Hi There, 

I'm new on Foundation-apps, Creating new app and install dependencies all are fine but when I gulp(build) the project it gives me some errors and this css file " assets/css/app.css " is missing. 

here are errors while building project.

![image](https://cloud.githubusercontent.com/assets/8864393/6381263/c36261b8-bd0d-11e4-86cb-03734030e096.png)

**Comment by gakimball** [comment]
@fhassan2014 The template project uses Bundler to install the right version of Sass for you. Run `bundle install` and then try compiling again.

The next version of the CLI won't require Ruby or Bundler, thankfully :)

**Comment by fhassan2014** [comment]
@gakimball here I tested again. looks fine your test build:

![image](https://cloud.githubusercontent.com/assets/8864393/6381339/6b20f54a-bd0e-11e4-9bf7-cf1d9fb330f6.png)

**Comment by gakimball** [comment]
@fhassan2014 Ah that's fantastic :) Thanks again for trying it out.

**Comment by fhassan2014** [comment]
If I go your test build and type foundation-apps watch. Nothing happened. should I run gulp or something to see on localhost.

![image](https://cloud.githubusercontent.com/assets/8864393/6381564/06e299b0-bd10-11e4-959c-b6cc8e324b04.png)

**Comment by gakimball** [comment]
@fhassan2014 You can just run `gulp` if you have it installed globally. The way the `watch` and `build` commands work changed slightly in the new CLI, although the functionality is the same.

`foundation-apps watch` is identical to `npm start` which is identical to `gulp`.

**Comment by fhassan2014** [comment]
After installing bundle: same error

![image](https://cloud.githubusercontent.com/assets/8864393/6381715/61d1f414-bd11-11e4-804e-01a6815f8e42.png)

**Comment by fhassan2014** [comment]
Hi @gakimball getting these errors while watch command. Any suggestion would be appreciated. Need to test one project to see how Foundation for apps works. Thanks,

**Comment by gakimball** [comment]
@fhassan2014 Try just running `gulp` inside your project folder instead, since that's what the `watch` command is running. (If you don't have Gulp: `npm install -g gulp`)

The `watch` command will make more sense once all three of the Foundation frameworks use the CLI, and then `watch` becomes a shorthand for "run whatever this project needs to run".

**Comment by soumak77** [comment]
@gakimball Just as curiosity, what are those three frameworks?  Foundation for Apps, Foundation for Sites, and ...?

**Comment by gakimball** [comment]
@soumak77 [Ink](http://zurb.com/ink/) is the third one, which is being rebranded as Foundation for Emails when v2.0 comes out in a few months. Unifying the branding partially represents our push to share more code, styles, and best practices between the three frameworks.

**Comment by soumak77** [comment]
@gakimball Oh ok, yeah I new about Ink and the rebranding to Foundation for Emails, but didn't realize there was a need for a CLI for that framework.  I thought it was just templates you copied and manually injected whatever content you needed into the provided areas.  I haven't used F4E yet (which may be clear by my last statement), though that is definitely something I want to get started with as my company emails need some much loving care.

**Comment by fhassan2014** [comment]
@gakimball now getting this final page. Not sure it suppose to be like this. 
![result](https://cloud.githubusercontent.com/assets/8864393/6398770/e1d7479a-bdbb-11e4-9221-a1b89e343dd5.jpg)

and got this error before building project:
![image](https://cloud.githubusercontent.com/assets/8864393/6398790/0f17a376-bdbc-11e4-8b52-ea470e75f0b7.png)

Before all this I had this Warning Engin: foundation-cli@1.0.4: wanted: <"node":">=0.10.33"> <current: 0.10.32, npm: 1.4.28

I hope you'll find any solution on this. I also tried running gulp and gave me this screen: 
![image](https://cloud.githubusercontent.com/assets/8864393/6398861/8ee3701c-bdbc-11e4-85d5-805635101a3c.png)

**Comment by gakimball** [comment]
@fhassan2014 The second error `EADDRINUSE` means you already have a server for another Apps project running, or at least you have something else running on port 8080. Close out your other build process, run `gulp` again, and then it should work.

**Comment by fhassan2014** [comment]
@gakimball Thanks! it's fine, the only thing lift is browser is looking these missing files. any thoughts.

![result](https://cloud.githubusercontent.com/assets/8864393/6398998/b1177916-bdbd-11e4-9650-8d4d6518577f.jpg)

**Comment by gakimball** [comment]
@fhassan2014 It looks like the Sass isn't compiling to CSS; that's why there's a "Not Found" error in the console, for `css/app.css`. Are you still getting the same `gulp-ruby-sass` error in your other screenshots?

**Comment by fhassan2014** [comment]
@gakimball Yes.

**Comment by gakimball** [comment]
@fhassan2014 I just noticed you're running a pre-release build of JRuby, so I think that's the issue. The error is being generated by the Sass executable, meaning it's a Ruby error, not a JavaScript one. You might need to switch to a stable version of JRuby, or file an issue with their team.

**Comment by gakimball** [comment]
This is a somewhat older issue, so I'm going to close it out. The original issue filed has been fixed, and version 1.1 of the CLI, which we published last month, greatly improves the stability and error-handling of the project creation process.

If y'alls have any more issues with the CLI in the future, do let us know :)

**Comment by narration** [comment]
Thanks, Geoff.
